src/main/java/seedu/address/ui/applicant/ApplicantCard.java

```diff
@@ -50,21 +54,13 @@ public ApplicantCard(Applicant applicant, int displayedIndex) {
         this.applicant = applicant;
```

Comment:
Java docs for this method seems outdated. May want to change it.

Reply:
Fixed!